Documentation/mender-ci-cd.md

```diff
@@ -5,6 +5,59 @@ CI/CD setup using GitLab on Google Cloud Platform (GCP). For a basic introductio
 to GitLab, refer to the dedicated guide in `gitlab-ci.md`.
 
 
+## GitLab's master runner dependencies
+
+We have only one permanent machine in GCP. It receives requests from GitLab for running jobs and
+then launches, and configures "slaves" to perform the actual CI job.
+
+This machine requires three pieces of software:
+* `gitlab-runner` to communicate with GitLab backend
+* `docker-machine` to launch the slaves in GCP and install/configure Docker in them
```

Comment:
While we're at it, we could take the opportunity to modernize our language and say "workers" (2 instances). 🙂